server/chat.ts

```diff
@@ -548,6 +568,10 @@ export class CommandContext extends MessageContext {
 		try {
 			result = commandHandler.call(this, this.target, this.room, this.user, this.connection, this.cmd, this.message);
 		} catch (err) {
+			if (err.name?.endsWith('ErrorMessage')) {
```

Comment:
A comment about why we have to do this (ie hotpatching) would be helpful

Reply:
I think that'd be appropriate if it was `name === 'ErrorMessage'`, but this seems clearly intended to match all errors whose class names end with `ErrorMessage`. A comment would just do the same thing this code does.